src/coreclr/runtime.proj

```diff
@@ -17,6 +17,7 @@
       <_CoreClrBuildArg Condition="$([MSBuild]::IsOsPlatform(Windows)) and
                                    ('$(TargetArchitecture)' == 'x86' or '$(TargetArchitecture)' == 'x64') and
                                    '$(Configuration)' == 'Release' and
+                                   '$(ClrRuntimeSubset)' == 'true' and
```

Comment:
cc @trylek This is fixing bogus pgocheck error that one gets when skipping build of runtime subset.  Introduced by #46019